README.md

```diff
@@ -73,6 +73,9 @@ This will build the docker image as `prometheuscommunity/postgres_exporter:${bra
 * `exclude-databases`
   A list of databases to remove when autoDiscoverDatabases is enabled.
 
+* `allow-databases`
```

Comment:
I prefer `include` as the opposite of `exclude`.

Reply:
fixed